aws/tsconfig.json

```diff
@@ -1,21 +1,9 @@
 {
+    "extends": "@tsconfig/node14/tsconfig.json",
     "compilerOptions": {
-        "outDir": "bin",
-        "target": "es2016",
-        "module": "commonjs",
-        "moduleResolution": "node",
-        "declaration": true,
-        "sourceMap": true,
-        "stripInternal": true,
-        "experimentalDecorators": true,
-        "pretty": true,
-        "noFallthroughCasesInSwitch": true,
-        "noImplicitAny": true,
```

Comment:
there are a few configurations that would be nice to keep, I imagine you have some specific reason for changing the build so drastically in the context of this PR?

Reply:
Which configuration would you want to keep? For reference, this is the final configuration when using the preset:

```json
{
  "$schema": "https://json.schemastore.org/tsconfig",
  "display": "Node 14",

  "compilerOptions": {
    "outDir": "bin",
    "target": "es2020",
    "module": "esnext",
    "moduleResolution": "Node",
    "lib": ["es2020"],
    "strict": true,
    "esModuleInterop": true,
    "skipLibCheck": true,
    "forceConsistentCasingInFileNames": true,
  }
}
```

Some of the configurations are not relevant for us (such as `declaration`, `sourceMap`,  and `stripInternal`) because we are not publishing a package those are good for this scenario. For others, we are just re-declaring the defaults (`pretty`, `noImplicitAny and `strictNullChecks` when using `strict: true`).

I've re-add some options: 3f3bcce40bf01928bc67dc144be881126ac09a7a.

Reply:
I'm good with the new version 👍 
Not to be tackled in the context of this PR, but, is there any specific version to have two completely separate compilation units?